config/routes.rb

```diff
@@ -15,110 +17,110 @@
 
   as :user do
     # Sessions
-    get    '/login'  => 'users/sessions#new', :as => :new_user_session
-    post   '/login'  => 'users/sessions#create'
-    get    '/user'   => 'users/sessions#show', :as => :user_session
-    delete '/logout' => 'users/sessions#destroy', :as => :destroy_user_session
+    get    "/login"  => "users/sessions#new", as: :new_user_session
+    post   "/login"  => "users/sessions#create"
+    get    "/user"   => "users/sessions#show", as: :user_session
+    delete "/logout", to: "users/sessions#destroy", as: :destroy_user_session
 
     # Registrations
-    controller 'users/registrations' do
-      get  '/signup', action: :new, as: :new_user_registration
-      post '/signup', action: :create, as: :user_registration
+    controller "users/registrations" do
+      get  "/signup", action: :new, as: :new_user_registration
+      post "/signup", action: :create, as: :user_registration
     end
 
-    get '/dashboard' => 'dashboard#index', as: :dashboard
-    get '/dashboard/:filter' => 'dashboard#index', as: :dashboard_filter
+    get "/dashboard", to: "dashboard#index", as: :dashboard
+    get "/dashboard/:filter", to: "dashboard#index", as: :dashboard_filter
 
     # settings
-    get    '/settings' => 'users#edit', as: :edit_user
-    patch  '/settings' => 'users/registrations#update'
-    get    '/settings/email' => 'users/registrations#edit_email', :as => :edit_user_email
-    get    '/settings/profile' => 'users/profiles#edit', :as => :edit_user_profile
-    get    '/settings/notifications' => "users/notifications#edit", :as => :settings_notifications
-    patch  '/settings/notifications' => "users/notifications#update"
-
-    namespace :users, path: 'user' do
+    get    "/settings", to: "users#edit", as: :edit_user
+    patch  "/settings", to: "users/registrations#update"
+    get    "/settings/email", to: "users/registrations#edit_email", as: :edit_user_email
+    get    "/settings/profile", to: "users/profiles#edit", as: :edit_user_profile
+    get    "/settings/notifications" => "users/notifications#edit", as: :settings_notifications
+    patch  "/settings/notifications" => "users/notifications#update"
+
+    namespace :users, path: "user" do
       resource :balance, only: [:show] { post :withdraw }
       resource :payment_option, only: [:show, :create, :update]
       resource :tax_info, only: [:show, :create, :update] do
-        get ':form_type' => 'tax_infos#show'
+        get ":form_type", to: "tax_infos#show"
       end
     end
 
     # Confirmation
-    get    '/users/confirmation/new' => 'users/confirmations#new', :as => :new_user_confirmation
-    get    '/users/confirmation' => 'users/confirmations#show', :as => :user_confirmation
-    post   '/users/confirmation' => 'users/confirmations#create'
+    get    "/users/confirmation/new", to: "users/confirmations#new", as: :new_user_confirmation
+    get    "/users/confirmation", to: "users/confirmations#show", as: :user_confirmation
```

Comment:
Line is too long. [88/80]<br>Unnecessary spacing detected.<br>Put one space between the method name and the first argument.